package.json

```diff
@@ -20,7 +20,7 @@
     "@actions/github": "^4.0.0",
     "@octokit/types": "^6.14.2",
     "@octokit/webhooks": "^9.0.0",
-    "@types/node": "^14.14.37",
+    "@types/node": "^15.12.2",
```

Comment:
Could you change this to use a `16.x.x` version?

Reply:
i think that's that latest version?

Reply:
Huh so it is, how weird